### PR TITLE
chore: config deployer to deploy to generic network

### LINKED ISSUE #28: chore: config deployer to deploy to generic network
Currently, the deployer takes files from hardcoded paths

# Changes
- Take deploy files as input in `AlignedLayerDeployer`
- Upload eigenlayer config for [holesky](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/script/configs/holesky/Holesky_current_deployment.config.json)
-  Add deploy script without anvil

### PR DESCRIPTION
**Pull request by JuArce**
# Changes
- Add a new target `deploy-aligned-contracts` to deploy contracts
It takes the config from a .env file.

- Add Holesky deployment info for eigenlayer

- I have to use --slow flag for forge because on deploy the process halts waiting for tx confirmations

### CODE REVIEW

File: contracts/scripts/.env.example.holesky [hunk 1]
```diff
@@ -0,0 +1,5 @@
+RPC_URL=<rpc_url>
+PRIVATE_KEY=<private_key>
+EXISTING_DEPLOYMENT_INFO_PATH=./script/output/holesky/Holesky_current_deployment.config.json
+DEPLOY_CONFIG_PATH=./script/deploy/config/holesky/aligned.holesky.config.json
+OUTPUT_PATH=./script/output/holesky/alignedlayer_deployment_output.json
```
Comment (taturosati) [inline contracts/scripts/.env.example.holesky]: Missing newline

File: README.md [hunk 2]
```diff
@@ -89,6 +89,29 @@ This will send a dummy task to the ServiceManager. Make sure to have the Service
 
 The plonk proof is located at `task_sender/test_examples`.
 
+## Deployment
+
+To deploy the contracts to Testnet/Mainnet, you will need to set environment variables
+in a .env file in the same directory as the deployment script (`contracts/scripts/`).
```
Comment (taturosati) [inline README.md:95]: Specify here that you need to set addresses on `contracts/script/deploy/config/holesky/aligned.holesky.config.json` as well

**Review by taturosati — CHANGES_REQUESTED** [review]
🚧👍

**Review by MauroToscano — APPROVED** [review]

**Review by taturosati — APPROVED** [review]
lgtm

**Pull request merged**